runner/payload/transferonly/worker.go

```diff
@@ -226,8 +240,25 @@ func (t *transferOnlyPayloadWorker) sendTxs(ctx context.Context) error {
 	txs := make([]*types.Transaction, 0, numAccounts)
 	acctIdx := 0
 
+	randomInt := rand.Uint64()
+
 	for gasUsed < (t.params.GasLimit - 100_000) {
-		transferTx, err := t.createTransferTx(t.privateKeys[acctIdx], t.nextNonce[t.addresses[acctIdx]], t.addresses[(acctIdx+1)%numAccounts], big.NewInt(1))
+
+		dest := t.addresses[(acctIdx+1)%numAccounts]
+		if t.payloadParams.CreateAccounts != nil && *t.payloadParams.CreateAccounts {
+			fakeAddressOffset := t.currFakeAddr
+			t.currFakeAddr++
+
+			var addr common.Address
+			binary.BigEndian.PutUint64(addr[:], fakeAddressOffset)
+
+			binary.BigEndian.PutUint64(addr[8:], randomInt)
+			addr[19] = 0xff
+			addr[18] = 0xff
```

Comment:
Are `addr[16]` and `addr[17]` intentionally empty?

Reply:
Yeah, we don't really need the 18/19 either, but it helps ensure it's not an existing address. There's always a 64-bit random int per run and a 64-bit sequence number on the address for every transfer to ensure uniqueness as well.